src/gui/torrentcontenttreeview.cpp

```diff
@@ -106,6 +108,21 @@ void TorrentContentTreeView::keyPressEvent(QKeyEvent *event)
     }
 }
 
+void TorrentContentTreeView::mouseDoubleClickEvent(QMouseEvent *event)
+{
+    const QModelIndex index = indexAt(event->pos());
+    if (index.isValid())
```

Comment:
Wouldn't it make sense to check the column as well, so as not to apply the logic below to the unsuitable ones?